tools/drawablehelper.py

```diff
@@ -248,18 +248,44 @@ def center_drawable_to_models(drawable_obj: bpy.types.Object):
     model_objs = [
         child for child in drawable_obj.children if child.sollum_type == SollumType.DRAWABLE_MODEL]
 
-    center = Vector()
+    center = get_bounding_box_center_of_selected()
 
-    for obj in model_objs:
-        center += obj.location
-
-    center /= len(model_objs)
+    if center is None:
+        return
 
     drawable_obj.location = center
 
     for obj in model_objs:
         obj.location -= center
 
+def get_bounding_box_center_of_selected():
```

Comment:
Same-ish function as in boundhelper.py? Don't duplicate code. It could be moved to blenderhelper.py and be imported it by these two modules.

Reply:
Yes, will move it and remove the duplicated code